inc/project.class.php

```diff
@@ -1409,12 +1438,33 @@ function showForm($ID, $options = []) {
       echo "</td>";
       echo "<td>".__('Percent done')."</td>";
       echo "<td>";
-      Dropdown::showNumber("percent_done", ['value' => $this->fields['percent_done'],
-                                                 'min'   => 0,
-                                                 'max'   => 100,
-                                                 'step'  => 5,
-                                                 'unit'  => '%']);
-      echo "</td>";
+      $percent_done_params = [
+         'value' => $this->fields['percent_done'],
+         'min'   => 0,
+         'max'   => 100,
+         'step'  => 5,
+         'unit'  => '%'
+      ];
+      if ($this->fields['auto_percent_done']) {
+         $percent_done_params['specific_tags'] = ['disabled' => 'disabled'];
+      }
+      Dropdown::showNumber("percent_done", $percent_done_params);
+      $auto_percent_done_params = [
+         'type'      => 'checkbox',
+         'name'      => 'auto_percent_done',
+         'title'     => __('Automatically calculate'),
+         'onclick'   => "$(\"select[name='percent_done']\").prop('disabled') = !$(\"input[name='auto_percent_done']\").prop('checked');"
+      ];
+      if ($this->fields['auto_percent_done']) {
+         $auto_percent_done_params['checked'] = 'checked';
+      }
+      $checkbox_id = "check_".mt_rand();
+      Html::showCheckbox($auto_percent_done_params, $checkbox_id);
+      echo "<span class='very_small_space'>";
+      Html::showToolTip(__('Percentage is computed based on the average of all child project and task percent done'), [
+         'applyto'   => $checkbox_id
```

Comment:
This does not work, as the "real" checkbox is not displayed. I guess displaying the `i` icon is the best option here.